spring-cloud-gateway-core/src/test/resources/application.yml

```diff
@@ -7,6 +7,11 @@ test:
 spring:
   cloud:
     gateway:
+
+      httpclient:
+        connect-timeout: 100
```

Comment:
@ryanjbaxter i suppose we can not fix this without breaking anyone, but it bothers me that response timeout has units (ms) and connect timeout does not